.travis.yml

```diff
@@ -144,6 +144,25 @@ script:
     fi
   # different builds with CMake
   - if [ "$BUILD_MODE" == "cmake" ]; then
+
+      DESIRED_CMAKE_VERSION=3.16.3;
+      DEFAULT_CMAKE_VERSION="`cmake --version|sed -n '1s/^.* //p'`";
+      $PIP install cmake==$DESIRED_CMAKE_VERSION;
```

Comment:
we should use `$PIP install --upgrade pip` before installing cmake to avoid `ImportError: No module named 'skbuild'`.

Reply:
Thanks!